src/Test/Components.test.js

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import '@testing-library/jest-dom/extend-expect';
```

Comment:
- [ ] Kindly add tests to simulate user interactions, like firing events and so on.

  You visit [this tutorial](https://www.robinwieruch.de/react-testing-library/) for ideas.